content/concepts/webhooks.textile

```diff
@@ -37,53 +44,68 @@ __WebHooks are the HTTP APIs built by the vendors and registered at the company
 
 h2(#webhooks-use-case). Webhooks use case - a deeper dive
 
-There is a very small difference between traditional APIs and webhooks. API calls works on the request-based output mechanism, while webhooks work on event-based output mechanisms. In other words, unlike an API, which relies on user-input, webhooks are automated calls from example.com to server.
+Webhooks are important when it comes to establishing an event-based communication channel between your application and your service provider's application infrastructure. Webhooks keep key data in sync between two remote applications. 
+
+As a result, webhooks form the basis of the infrastructure that supports many online activities we take for granted. Take payment notifications. If an e-commerce retailer uses a third-party payment gateway, the ‘payment’ event happens in an environment external to the retailer’s website. Through webhooks, the payment gateway will call the retailer’s register API to provide the payment data as soon as it’s available. The retailer's server then accepts this data, enabling it to update its database and user-facing screens.
 
-Webhooks are important to establish the event-based communication channel between vendors and their service provider's application infrastructure. Webhooks keep two remote applications in-sync in terms of common data dependencies. With this, webhooks form the basis of the infrastructure that supports online activities we take for granted. Take the common example of payment notifications. If the e-commerce vendor gets the payment via Payment gateways, the event happens in an environment external to the vendor’s website. Through webhooks, the payment gateway will call the vendor’s register API with the payment data, and the vendor's server then accepts this data to update the database and dashboard.
+A number of high-profile service providers deliver key capabilities through the use of webhooks, which you can leverage. Let’s look at a couple of examples.
 
 h2(#webhooks-examples). Webhooks - real-world examples
 
 h3. 1. Marketing platforms
 
-Mailchimp is one of many platforms that relies on webhooks for everyday events such as subscribing, unsubscribing and profile changes. This is useful if you’re using MailChimp to signup users from your website to your newsletter but you also wanted to pass that data to your CRM system. SendGrid also offers webhooks for informing users when transactional emails are delivered, opened or unintentionally not delivered to the recipient.
+MailChimp is one of many platforms that relies on webhooks to support everyday tasks, such as subscribing to and unsubscribing from to email lists, as well as user profile changes. This is useful if you’re using MailChimp as the mechanism to sign website users up to your newsletter, but you also want to pass the data to your CRM system. 
```

Comment:
`unsubscribing from to email lists,` => `unsubscribing from email lists` I think?